### PR TITLE
Add components docs to zedkube module

### PR DESCRIPTION
**Pull request by naiming-zededa**
- added documents to zedkube.md in pillar/docs

### CODE REVIEW

File: pkg/pillar/docs/zedkube.md
Comment (zedi-pramodh) [inline pkg/pillar/docs/zedkube.md:13]: Do you mean  /run/zedkube/vmiVNC.run ?  Why would you put in /usr

**Review by zedi-pramodh — APPROVED** [review]
LGTM, after fixing the comment, please merge

File: pkg/pillar/docs/zedkube.md
Comment (naiming-zededa) [reply pkg/pillar/docs/zedkube.md]: fixed. thanks.

**Pull request merged**
